pages/querying/expressions.mdx

```diff
@@ -55,13 +55,32 @@ but this is supported:
 CREATE (n $propertyMap) RETURN n;
 ```
 
+If you want to filer a node based on a specific property, you can do it with the next query:
+
+```cypher
+MATCH (n) WHERE n.$propertyValue = $propertyDesiredValue RETURN n;
+```
+
+Using parameters with labels is also supported. To filter nodes based on a specific label, you can run the next query:
```

Comment:
Should we point out to the user that he can use labels in all types of clauses since he can't do that with properties?